charts/consul/templates/connect-inject-clusterrole.yaml

```diff
@@ -116,4 +116,232 @@ rules:
   verbs:
   - use
 {{- end }}
+- apiGroups:
+  - apps
+  resources:
+  - deployments
```

Comment:
Can we drop these resources until they're needed in the gateway controller to make the permissions a bit easier to review?